### PR TITLE
Convert account tool to cli package

### PR DESCRIPTION
**Pull request by JNProtzman**

### CODE REVIEW

**Comment by JNProtzman** [comment]
Let me know what you think of this library - if you're cool with it, I'll convert the remaining packages to it and eventually merge them all into the same CLI package. This is one path forward on creating the console we talked about.

**Comment by dcrodman** [comment]
How's this for a timely review?

Launching archon and then selecting server seems to cause the process to exit with no message. The main server executable also seems to have stopped responding to signals so you can't ctrl-c it. Was hoping to tinker around with this a little bit

File: Makefile [hunk 1]
```diff
@@ -20,8 +24,12 @@ test:
 protos:
 	./gen_protos.sh
 
-run: build
-	${BIN_DIR}/server -config ${CONFIG_PATH}
+setup:
+	./setup/setup.sh
+
+# cd first so we're in the same dir as the config file
+run: setup
```
Comment (JNProtzman) [inline Makefile:31]: If you run `make run`, then you'll automatically get launched into the GUI.

File: cmd/menu.go [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+	"github.com/jroimartin/gocui"
+	"github.com/urfave/cli/v2"
+)
+
+var menuItems = []string{
+	Archon.String(),
+	//Patcher.String(),
+	//GenerateServerCertificates.String(),
+	//RunPacketAnalyzer.String(),
+	//RunAccountTool.String(),
```
Comment (JNProtzman) [inline cmd/menu.go:18]: These don't work for two reasons:
- they need to be converted to the CLI package
- the ones that take user input will take some effort to get the GUI to accept stdin

I do have them all converted but they'll fail.

**Comment by JNProtzman** [comment]
What are you envisioning being able to run in the GUI? It's been a while since we've talked about it.

**Pull request closed without merging**